package pontointeligente.infrastructure.exception

object MessagesValidationsErrors {

    val COMPANY_ALREADY_EXISTS_ERROR: String = "company.already.exists.error"
    val COMPANY_DOES_NOT_EXISTS_ERROR: String = "company.does.not.exists.error"
    val EMPLOYEE_ALREADY_EXISTS_ERROR: String = "employee.already.exists.error"
    val EMPLOYEE_DOES_NOT_EXISTS_ERROR: String = "employee.does.not.exists.error"
    val LAUNCH_DOES_NOT_EXISTS_ERROR: String = "launch.does.not.exists.error"
    val LAUNCH_POINT_ALREADY_EXISTS_ERROR: String = "launch.point.already.exists.error"

    val OPERATION_NOT_ALLOWED: String = "operation.not.allowed"

    val INVALID_OPERATION: String = "invalid.operation"


}